app/assets/stylesheets/refills/_hero.scss

```diff
@@ -0,0 +1,64 @@
+.hero {  
+  ///////////////////////////////////////////////////////////////////////////////////
+  $base-border-radius: 3px !default;
+  $base-accent-color: #477DCA !default;
+  $large-screen: em(860) !default;
+  //////////////////////////////////////////////////////////////////////////////////
+
+  $hero-background-top: #7F99BE; 
+  $hero-background-bottom: #20392B;
+  $hero-color: red;
+  $gradient-angle: 10deg;
+  $hero-image: asset_path('hero-doge');
```

Comment:
Name of function `asset_path` should be written in all lowercase letters with hyphens instead of underscores<br>Prefer double-quoted strings